package.json

```
@@ -0,0 +1,77 @@
+{
+    "name": "eversign",
+    "label": "Eversign Package",
+    "dependencies": [
+        {
+            "type": "service",
+            "name": "http",
+            "version": "v1.2.6",
```

```suggestion
            "version": "v1.3.7",
```
Let's just update this, if someone creates a new dedicated service the validation may fail because of not matching the version.